application/app/views/layouts/_repo_resolver_bar.html.erb

```diff
@@ -1,11 +1,14 @@
 <%
   id = local_assigns.fetch(:id, 'repo-resolver-bar')
+  input_id = "#{id}_input"
+  color = local_assigns.fetch(:color, 'bg-primary-subtle')
+  collapse = local_assigns.fetch(:collapse, false)
   show_images = local_assigns.fetch(:show_images, true)
```

Comment:
I think if we are making this bar always display, we should remove the Explore link at the top navigation bar and update the docs here https://github.com/IQSS/ondemand-loop/blob/main/docs/guide/content/user_guide/index.md?plain=1#L37 and https://github.com/IQSS/ondemand-loop/blob/main/docs/guide/content/user_guide/finding_data.md?plain=1#L43

Reply:
Fixed